scenarios/reproducers/validated-architecture-shiftstack.yml

```diff
@@ -0,0 +1,159 @@
+# Please note, all paths are on the controller-0, meaning managed by the
+# Framework. Please do not edit them!
+_arch_repo: "/home/zuul/src/github.com/openstack-k8s-operators/architecture"
+cifmw_architecture_scenario: hci
+cifmw_ceph_client_vars: /tmp/ceph_client.yml
+cifmw_ceph_client_values_post_ceph_path_src: >-
+  {{ _arch_repo }}/examples/va/hci/values.yaml
+cifmw_ceph_client_values_post_ceph_path_dst: >-
+  {{ cifmw_ceph_client_values_post_ceph_path_src }}
+cifmw_ceph_client_service_values_post_ceph_path_src: >-
+  {{ _arch_repo }}/examples/va/hci/service-values.yaml
+cifmw_ceph_client_service_values_post_ceph_path_dst: >-
+  {{ cifmw_ceph_client_service_values_post_ceph_path_src }}
+
+# HERE if you want to overload kustomization, you can uncomment this parameter
+# and push the data structure you want to apply.
+# cifmw_architecture_user_kustomize:
+#   stage_0:
+#     'network-values':
+#       data:
+#         starwars: Obiwan
+
+# HERE, if you want to stop the deployment loop at any stage, you can uncomment
+# the following parameter and update the value to match the stage you want to
+# reach. Known stages are:
+#   pre_kustomize_stage_INDEX
+#   pre_apply_stage_INDEX
+#   post_apply_stage_INDEX
+#
+# cifmw_deploy_architecture_stopper:
+
+# What about some tempest?
+cifmw_run_tests: true
+cifmw_run_tempest: true
+cifmw_tempest_tests_allowed:
+  - tempest.scenario.test_network_basic_ops.TestNetworkBasicOps
+
+# This will instruct libvirt_manager to create 3 compute and
+# the ansible-controller, consuming the networks created and
+# managed by devscripts.
+# Note that the "osp_trunk" network is the equivalent of the
+# "private_net" in the CI layout, and will hold the VLAN for
+# network isolation.
+# The "ocpbm" network is managed by devscripts as well, and
+# provides access to Internet. This will be the equivalent of the
+# "public network" as seen in CI.
+cifmw_use_libvirt: true
+cifmw_libvirt_manager_configuration:
+  networks:
+    osp_trunk: |
+      <network>
+        <name>osp_trunk</name>
+        <forward mode='nat'/>
+        <bridge name='osp_trunk' stp='on' delay='0'/>
+        <ip family='ipv4'
+        address='{{ cifmw_networking_definition.networks.ctlplane.network |
+                    ansible.utils.nthhost(1) }}'
+        prefix='{{ cifmw_networking_definition.networks.ctlplane.network |
+                   ansible.utils.ipaddr('prefix') }}'>
+        </ip>
+      </network>
+  vms:
+    compute:
+      root_part_id: >-
+        {{
+          (cifmw_repo_setup_os_release is defined and cifmw_repo_setup_os_release == 'rhel') |
+          ternary(4, 1)
+        }}
+      amount: "{{ [cifmw_libvirt_manager_compute_amount|int, 3] | max }}"
+      image_url: "{{ cifmw_discovered_image_url }}"
+      sha256_image_name: "{{ cifmw_discovered_hash }}"
+      image_local_dir: "{{ cifmw_basedir }}/images/"
+      disk_file_name: "base-os.qcow2"
+      disksize: 200
+      memory: 50
+      cpus: 8
```

Comment:
different from default validated-architecture-1.yml

Reply:
if CPU is the only difference, we should consider doing the same thing done regarding the compute amount and allow passing it as an inline parameter

Reply:
it's CPU, memory and disk. I'm creating the variables for these 3. Moved these vars to shift-on-stack-overrides.yml as suggested